Jellyfin.Server/Program.cs

```diff
@@ -177,15 +177,25 @@ private static ServerApplicationPaths CreateApplicationPaths(StartupOptions opti
                 }
                 else
                 {
-                    // Let BaseApplicationPaths set up the default value
-                    configDir = null;
-                }
-            }
+                    if (RuntimeInformation.IsOSPlatform(OSPlatform.Windows))
+                    {
+                        configDir = Environment.GetFolderPath(Environment.SpecialFolder.ApplicationData);
+                    }
+                    else
+                    {
+                        // $XDG_CONFIG_HOME defines the base directory relative to which user specific configuration files should be stored.
+                        configDir = Environment.GetEnvironmentVariable("XDG_CONFIG_HOME");
+                        // If $XDG_CONFIG_HOME is either not set or empty, $HOME/.config should be used.
```

Comment:
This comment doesn't match the line below when going to `~/.local/share`, if I understand correctly.